web/src/cypress/integration/profile.ts

```diff
@@ -99,6 +99,27 @@ function testProfile(): void {
       cy.get('body').should('contain', `${name} (${type})`)
     })
 
+    it('should return error with link to conflicting user', () => {
+      cy.fixture('profile').then((prof) => {
+        cy.addContactMethod({ userID: prof.id }).then(
+          (contactMethod: ContactMethod) => {
+            cy.pageFab('Add Contact Method')
+            cy.dialogTitle('Create New Contact Method')
+            cy.dialogForm({
+              name: c.word({ length: 8 }),
+              type: contactMethod.type,
+              value: contactMethod.value,
+            })
+            cy.dialogClick('Submit')
+            cy.dialogContains(
+              'Contact method already exists for that type and value: ' +
+                prof.name,
+            ).click()
+          },
```

Comment:
We should assert that it takes the user to the correct page, or at least assert on the link address (instead of clicking)